components/EditUpdateForm.js

```diff
@@ -205,6 +207,33 @@ class EditUpdateForm extends React.Component {
               />
             </Container>
           </div>
+
+          <div className="row">
+            <div className="col large">
+              <Container mb={2} fontWeight="500" fontSize="1.6rem" lineHeight="1.7">
+                <Box as="span">
+                  <FormattedMessage
+                    id="update.makePublicOn.label"
+                    defaultMessage="Automatically make the update public on this date:"
+                  />
+                </Box>
+              </Container>
+              <StyledInputField htmlFor="makePublicOn">
+                {inputProps => (
+                  <StyledInput
+                    {...inputProps}
+                    type="date"
+                    value={update.makePublicOn ? moment(update.makePublicOn).format('YYYY-MM-DD') : ''}
+                    onChange={e => this.handleChange('makePublicOn', moment(e.target.value).toISOString())}
+                    width="100%"
+                    maxWidth="40em"
+                    placeHolder="Normal"
```

Comment:
What's the effect of this line?

Reply:
This line calls the handleChange method passing the date in ISO Format.

Reply:
Sorry, not the `onChange` but the `placeHolder="Normal"`

Reply:
I copied from the another component "title", but looking deeper into the StyledInput class, I didn't find any reference for this property.